Sources/UI/Crash Logging/CrashLoggingView.swift

```diff
@@ -97,14 +97,11 @@ extension CrashLoggingView {
         let error = SentryTestError(title: "Test Error")
 
         do {
-            try crashLogging.logErrorImmediately(error) { result in
-                switch result {
-                    case .success:
-                        sendErrorAndWaitStatus = .success
-                    case .failure(let err):
-                        sendingError = err
-                        sendErrorAndWaitStatus = .error
-                }
+            try crashLogging.logErrorImmediately(
+                error,
+                userInfo: ["custom-userInfo-key": "custom-userInfo-value"]
```

Comment:
I realized we never "tested" sending custom information, so I added it. Here's the result:

<img width="1023" alt="image" src="https://user-images.githubusercontent.com/1218433/196597311-505798ef-2558-4845-9c8c-37911bc5da9a.png">